internal/graphicscommand/image.go

```diff
@@ -190,25 +188,17 @@ func (i *Image) IsInvalidated() bool {
 	return i.image.IsInvalidated()
 }
 
-// Dump dumps the image to the specified path.
-// In the path, '*' is replaced with the image's ID.
+// dumpTo dumps the image to the specified writer.
 //
 // If blackbg is true, any alpha values in the dumped image will be 255.
 //
 // This is for testing usage.
-func (i *Image) Dump(graphicsDriver graphicsdriver.Graphics, path string, blackbg bool, rect image.Rectangle) error {
+func (i *Image) dumpTo(graphicsDriver graphicsdriver.Graphics, blackbg bool, rect image.Rectangle, w io.Writer) error {
```

Comment:
Please move `w` to the first argument

Reply:
ah yeah, I forgot to do this earlier. good catch